LogMonitor/src/LogMonitor/Utility.h

```diff
@@ -45,7 +45,7 @@ class Utility final
         _In_ const std::wstring& To
     );
 
-    static bool isJsonNumber(_In_ PWSTR str);

```

Comment:
Any reason for this change? PWSTR is a Windows-specific type.

Reply:
The only place I was using this function, already had `wstring`, there was no need of the extra conversion which was ugly too. Easier to convert from PWSTR -> wstring.